build/lib/release.sh

```diff
@@ -208,11 +210,38 @@ function kube::release::package_server_tarballs() {
 
     # This fancy expression will expand to prepend a path
```

Comment:
the `mkdir -p "${release_stage}/addons` line probably belongs in `package_server_tarballs`

Reply:
done